<launch>
    <!-- load the urdf and srdf -->
<!--    <include file="$(find ur_description)/urdf/ur10_robotiq85_gripper.xacro"/>-->
<!--    <param name="robot_description_semantic" textfile="$(ur10_gripper_moveit_config)/config/ur10.srdf"/>-->

<!--    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>-->
<!--    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>-->

    <node name="test_collision_model" pkg="sbpl_collision_checking_test" type="test_collision_model" output="screen">
        <rosparam command="load" file="$(find sbpl_collision_checking_test)/config/collision_model_ur10.yaml"/>
    </node>
</launch>
